riemann/tests/helpers.py

```diff
@@ -276,23 +276,23 @@
                 ]
             },
         'ser': {
-            'version': bytes.fromhex('00000000'),
-            'locktime': bytes.fromhex('00000000'),
+            'version': bytes.fromhex('01000000'),
+            'locktime': bytes.fromhex('cafd0700'),
             'ins': [
                 {
                     'id': 0,
                     'hash': bytes.fromhex('d2941b532f6d3d54d596345b50972b3995983239884037a52aab799ec84292ee'),
                     'index': bytes.fromhex('00000000'),
-                    'value': bytes.fromhex(''),
-                    'pk_script': bytes.fromhex('0014758ce550380d964051086798d6546bebdca27a73'),
+                    'value': bytes.fromhex('c0d4010000000000'),
+                    'pk_script': bytes.fromhex('a914419fbecd4b8f90c779b773a0a9010dd25423b06b87'),
```

Comment:
Correct this back to `0014758ce550380d964051086798d6546bebdca27a73`

Reply:
(ouo) b